src/ui/pages/SendForm/shared/buildSolanaTransfer.ts

```diff
@@ -0,0 +1,105 @@
+import type { TransactionInstruction } from '@solana/web3.js';
+import {
+  Connection,
+  PublicKey,
+  Transaction,
+  SystemProgram,
+} from '@solana/web3.js';
+import {
+  getAssociatedTokenAddress,
+  createTransferInstruction,
+  getMint,
+  createAssociatedTokenAccountInstruction,
+} from '@solana/spl-token';
+import type { AddressPosition } from 'defi-sdk';
+import type { EmptyAddressPosition } from '@zeriontech/transactions';
+import { Networks } from 'src/modules/networks/Networks';
+import type { NetworkConfig } from 'src/modules/networks/NetworkConfig';
+import { getAddress } from 'src/modules/networks/asset';
+import { createChain } from 'src/modules/networks/Chain';
+import { invariant } from 'src/shared/invariant';
+import { commonToBase } from 'src/shared/units/convert';
+import type { SendFormState } from './SendFormState';
+
+export async function buildSolanaTransfer(
```

Comment:
why do we use build here instead of create, like we do with ethereum transactions?

Reply:
Ok, I see the point :)